addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -821,7 +821,7 @@ body.editor_enable.editor_has_snippets {
         we-button.o_we_user_value_widget {
             > div {
                 // Needed otherwise cannot work because of flex display
-                @include o-text-overflow(block);
+                @include o-text-overflow(flex);
```

Comment:
Did you read the comment just above the line you modify when coding and reviewing?